modules/actors/actor-sheet-ffg.js

```diff
@@ -101,6 +104,43 @@ export class ActorSheetFFG extends ActorSheet {
           data.data.stats.credits.value = data.data.stats.credits.value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
         }
         break;
+      case "vehicle":
+        // add the crew to the items of the vehicle
+        data.crew = [];
+        // look up the flag data
+        let crew = game.actors.get(data.actor._id).flags.starwarsffg.crew;
+        if (crew) {
+          for (let i = 0; i < crew.length; i++) {
+            // iterate over the crew members in the flag data
+            let actor = game.actors.get(crew[i].actor_id);
```

Comment:
```suggestion
            const actor = game.actors.get(crew[i].actor_id);
```
Missed one of our old immutable friends. 😂